### PR TITLE
Fixed ExtensionManagementUtility::extRelPath()

### PR DESCRIPTION
**Pull request by ttimpe**
Fixed the extension not loading in TYPO3 due to the removal of ExtensionManagementUtility::extRelPath()

### CODE REVIEW

**Comment by liayn** [comment]
Thanks for your PR. The master branch is not yet v10 compatible (see composer.json) hence no v10 adjustments have been made.
We need to coordinate with @nicodh when/whether we merge master branch into the 9.5 branch and start making master v10 compatible.

**Comment by simonschaufi** [comment]
This is actually already throwing deprecation warnings in TYPO3 9 and should be replaced with `extPath() in conjunction with PathUtility::getAbsoluteWebPath() instead.`. I'm not sure if this PR is actually correct.

File: Classes/ViewHelpers/Be/ConfigurationViewHelper.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public function render()
     {
         $this->pageRenderer = GeneralUtility::makeInstance(PageRenderer::class);
 
-        $baseUrl = '../' . ExtensionManagementUtility::siteRelPath('extension_builder');
+        $baseUrl = ExtensionManagementUtility::extPath('extension_builder');
         $this->pageRenderer->disableCompressJavascript();
         $this->pageRenderer->addJsFile($baseUrl . 'Resources/Public/jsDomainModeling/jquery.min.js');
```
Comment (liayn) [inline Classes/ViewHelpers/Be/ConfigurationViewHelper.php]: This should better use the "EXT:extension_builder/Resources..." path spec

File: Classes/Configuration/ExtensionBuilderConfigurationManager.php [hunk 2]
```diff
@@ -117,7 +117,7 @@ public function getSettings($typoscript = null)
         $settings = $typoscript['module.']['extension_builder.']['settings.'];
         $settings['extConf'] = $this->getExtensionBuilderSettings();
         if (empty($settings['publicResourcesPath'])) {
-            $settings['publicResourcesPath'] = ExtensionManagementUtility::siteRelPath('extension_builder') . 'Resources/Public/';
+            $settings['publicResourcesPath'] = ExtensionManagementUtility::extPath('extension_builder'). 'Resources/Public/';
```
Comment (liayn) [inline Classes/Configuration/ExtensionBuilderConfigurationManager.php]: This is an absolute path now, but was a relative path before. I'm not sure whether this change is safe.

**Comment by nicodh** [comment]
> We need to coordinate with @nicodh when/whether we merge master branch into the 9.5 branch and start making master v10 compatible.

I also talked to @ohader about the v10 compatibility step. Best solution would be to support both versions if it's not too much effort.

**Comment by nicodh** [comment]
I created a milestone so we can assign issues and get an overview what has to be done...

File: Classes/ViewHelpers/Be/ConfigurationViewHelper.php (hunk 1 above)
Comment (nicodh) [reply Classes/ViewHelpers/Be/ConfigurationViewHelper.php]: In fact extPath is not working here. Has to be surrounded with
/TYPO3\CMS\Core\Utility\PathUtility::getAbsoluteWebPath()

Comment (nicodh) [reply Classes/ViewHelpers/Be/ConfigurationViewHelper.php]: $baseUrl = PathUtility::getAbsoluteWebPath(ExtensionManagementUtility::extPath('extension_builder'));

**Comment by liayn** [comment]
I think this is fixed now already in the v10 branch

**Pull request closed without merging**